packages/kobotoolbox/src/Utils.js

```diff
@@ -15,7 +15,15 @@ export const prepareNextState = (state, response) => {
 };
 
 export async function request(state, method, path, opts) {
-  const { baseURL, apiVersion, username, password } = state.configuration;
+  const { baseUrl, apiVersion, username, password } = state.configuration;
+
+  let baseURL = baseUrl || state.configuration.baseURL;
+
+  if (!baseUrl) {
```

Comment:
This won't work because  if baseURL is set, the baseUrl var will have a value and this block won't be entered. Did you test it manually?

I think you should be more explicit and have a whole if statement around state.configuration.baseURL, and do the warning and assignment inside

Reply:
I did test it, but let me change

Reply:
Fixed